demo/src/screens/componentScreens/TextFieldScreen.tsx

```diff
@@ -16,6 +16,7 @@ import {
 import Assets from '../../assets/Assets';
 const {KeyboardAwareInsetsView} = Keyboard;
 const priceFormatter = Intl.NumberFormat('en-US');
+const validationIcon = require('../../assets/icons/star.png');
```

Comment:
Let's use a better icon for this, we want to make the example to make sense.. 
You can copy the icon from private for that matter..

Reply:
Done